10-std.tex

```diff
@@ -539,3 +539,145 @@ \section{Input/Output}
 
 \section{Sys/sys}
 \label{std-sys}
+
+\section{Remoting}
+\label{std-remoting}
+
+Haxe remoting is a way to talk between different platforms. With Haxe remoting we can communicate transparently, send data and call methods, between server and client side.
+
+\subsection{Remoting Connection}
+\label{std-remoting-connection}
+
+In order to use remoting, there must be a connection established first. There are two kinds of Haxe Remoting connections: 
+\begin{description}
+	\item[\expr{haxe.remoting.Connection}] is used for \emph{synchronous connections}, where the results can be directly obtained when calling a method. 
+	\item[\expr{haxe.remoting.AsyncConnection}] is used for \emph{asynchronous connections}, where the results are events that will happen later in the execution process.
+\end{description}
+
+\paragraph{Start a connection}
+Haxe remoting can be used to communicate across different platforms. There are some target-specific constructors with different purposes to set up a connection:
+
+\begin{description}
+	\item[All targets:]
+		\begin{description}
+			\item[\expr{HttpAsyncConnection.urlConnect(url:String)}]  
+				Returns an asynchronous connection to the given URL that can be a Haxe/Server application. 
+		\end{description}
+		
+	\item[Flash:]
+		\begin{description}
+			\item[\expr{ExternalConnection.jsConnect(name:String, ctx:Context)}]  
+				Allows a connection to the local JavaScript Haxe code. The JS Haxe code must be compiled with the class ExternalConnection included. This is only working with Flash Player 8 and higher.
```

Comment:
"This only works", the progressive "is working" is only used when it is actually happening right now this very moment.